scripts_ZA/compareShapesForTTbarEstim.py

```diff
@@ -0,0 +1,298 @@
+            histojet1pt_name = "jet1_pt_MuEl_hZA_lljj_deepCSV_btagM_"
+            histojet2pt_name = "jet2_pt_MuEl_hZA_lljj_deepCSV_btagM_"
+            histojjDR_name = "jj_DR_j_j_MuEl_hZA_lljj_deepCSV_btagM_"
+            histolep1pt_name = "lep1_pt_MuEl_hZA_lljj_deepCSV_btagM_"
+            histolep2pt_name = "lep2_pt_MuEl_hZA_lljj_deepCSV_btagM_"
+            histojjpt_name = "jj_pt_MuEl_hZA_lljj_deepCSV_btagM_"
+            if not str(split_filename[-1]).startswith('MuonEG'):
+                continue
+        elif not data:
+            histoMjj_name = "jj_M_MuMu_hZA_lljj_deepCSV_btagM_"
+            histoMlljj_name = "lljj_M_MuMu_hZA_lljj_deepCSV_btagM_"
+            histoMll_name = "ll_M_MuMu_hZA_lljj_deepCSV_btagM_"
+            histojet1pt_name = "jet1_pt_MuMu_hZA_lljj_deepCSV_btagM_"
+            histojet2pt_name = "jet2_pt_MuMu_hZA_lljj_deepCSV_btagM_"
+            histojjDR_name = "jj_DR_j_j_MuMu_hZA_lljj_deepCSV_btagM_"
+            histolep1pt_name = "lep1_pt_MuMu_hZA_lljj_deepCSV_btagM_"
+            histolep2pt_name = "lep2_pt_MuMu_hZA_lljj_deepCSV_btagM_"
+            histojjpt_name = "jj_pt_MuMu_hZA_lljj_deepCSV_btagM_"
+            if not str(split_filename[-1]).startswith('TT_TuneCUETP8M2T4'):
+                continue
+        f = ROOT.TFile(filename)
+        print filename
+        single_Mjj = f.Get(histoMjj_name)
+        single_Mjj.SetDirectory(0)
+        Mjj.Add(single_Mjj)
+
+        single_Mlljj = f.Get(histoMlljj_name)
+        single_Mlljj.SetDirectory(0)
+        Mlljj.Add(single_Mlljj)
+
+        single_Mll = f.Get(histoMll_name)
+        single_Mll.SetDirectory(0)
+        Mll.Add(single_Mll)
+
+        single_jet1pt = f.Get(histojet1pt_name)
+        single_jet1pt.SetDirectory(0)
+        jet1pt.Add(single_jet1pt)
+
+        single_jet2pt = f.Get(histojet2pt_name)
+        single_jet2pt.SetDirectory(0)
+        jet2pt.Add(single_jet2pt)
+
+        single_jjDR = f.Get(histojjDR_name)
+        single_jjDR.SetDirectory(0)
+        jjDR.Add(single_jjDR)
+
+        single_lep1pt = f.Get(histolep1pt_name)
+        single_lep1pt.SetDirectory(0)
+        lep1pt.Add(single_lep1pt)
+
+        single_lep2pt = f.Get(histolep2pt_name)
+        single_lep2pt.SetDirectory(0)
+        lep2pt.Add(single_lep2pt)
+
+        single_jjpt = f.Get(histojjpt_name)
+        single_jjpt.SetDirectory(0)
+        jjpt.Add(single_jjpt)
+
+    if not data:
+        Mjj.Scale(lumi)
+        Mlljj.Scale(lumi)
+        Mll.Scale(lumi)
+        jet1pt.Scale(lumi)
+        jet2pt.Scale(lumi)
+        jjDR.Scale(lumi)
+        lep1pt.Scale(lumi)
+        lep2pt.Scale(lumi)
+        jjpt.Scale(lumi)
+ 
+    list_histos.append(Mjj)
+    list_histos.append(Mlljj)
+    list_histos.append(Mll)
+    list_histos.append(jet1pt)
+    list_histos.append(jet2pt)
+    list_histos.append(jjDR)
+    list_histos.append(lep1pt)
+    list_histos.append(lep2pt)
+    list_histos.append(jjpt)
+    return list_histos
+
+
+def main():
+    list_histos_data = []
```

Comment:
overall suggestion for this script: rename `data` in `ttbar_from_data` or sth like that, it would be easier to read the script ?